pkg/container-utils/containerutils.go

```diff
@@ -83,7 +83,7 @@ var ContainerRuntimes = []string{
 	crio.Name,
```

Comment:
I think that Flatcar has both docker and containerd running.  Because of the order in this array, it would pick docker.

Reply:
It will not happen with the new approach.